### PR TITLE
Update TopOpt.jl GSoC project list

### PR DESCRIPTION
**Pull request by mohamed82008**
This PR updates the list of GSoC projects for the TopOpt project.

### CODE REVIEW

**Comment by mohamed82008** [comment]
@TheCedarPrince I saw your slack message so I'm pinging you. Please review this when you get a chance.

File: jsoc/gsoc/topopt.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
 
 The following is a tentative list of projects in topology optimisation that you could be working on in the coming Julia Season of Contributions or Google Summer of Code. If you are interested in exploring any of these topics or if you have other interests related to topology optimisation, please reach out to the main mentor [Mohamed Tarek](https://github.com/mohamed82008) via email.
 
+## Testing and benchmarking of TopOpt.jl
+
+**Project difficulty**: Easy to Medium
+
+**Work load**: 175 or 350 hours
```
Comment (TheCedarPrince) [inline jsoc/gsoc/topopt.md:11]: Hey @mohamed82008 , only one small concern here. I would highly advise making the executive decision of choosing either only Easy or Medium. If you feel that a project could potentially be difficult, I'd suggest just going with Medium.

With regards to the workload, I'd also suggest just stating only one time interval or another.

**Review by TheCedarPrince — COMMENTED** [review]
Left only one comment; otherwise, this looks great and will merge after you respond! 

Cheers! ~ tcp :deciduous_tree:

**Comment by TheCedarPrince** [comment]
@mohamed82008 -- have you gotten a chance to look at this yet?

File: jsoc/gsoc/topopt.md (hunk 1 above)
Comment (mohamed82008) [reply jsoc/gsoc/topopt.md]: done

**Comment by mohamed82008** [comment]
This is probably ready.

**Review by TheCedarPrince — APPROVED** [review]
Yup! Thanks @mohamed82008 -- merged in!

**Pull request merged**